.template-lintrc.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
 
 module.exports = {
-  extends: 'recommended'
+  extends: 'octane',
```

Comment:
We've actually deprecated the 'octane' config in v3 in favor of that config being merged into 'recommended'. Let's put this back.

Reply:
*cough*

https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/.template-lintrc.js#L4